2-projects/1-todo-list/script.js

```diff
@@ -1,8 +1,49 @@
+const list = document.getElementById('todo-list')
+const input = document.getElementById('todoInput');
+
+
 function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
+  
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+  list.innerHTML = '';
+  for (let currentList = 0; currentList < todos.length; currentList++) {
+    let newToDoList = document.createElement('li');
+    newToDoList.innerHTML = `<li
+    class="list-group-item d-flex justify-content-between align-items-center"
+  >
+    ${todos[currentList].task}
+    <span class="badge bg-primary rounded-pill">
+      <!-- each of these <i> tags will need an event listener when we create them in Javascript -->
+      <i class="fa fa-check" aria-hidden="true"></i>
+      <i class="fa fa-trash" aria-hidden="true"></i>
+    </span>
+  </li>`
+    list.appendChild(newToDoList);
+    let checkMarkBtn = document.querySelectorAll('.fa-check');
+    let delBtn = document.querySelectorAll('.fa-trash')
+    checkMarkBtn[currentList].addEventListener('click', () => {
+      newToDoList.classList.toggle('cross')
+    });
+    delBtn[currentList].addEventListener('click', () => {
+      newToDoList.classList.add('hidden')
+    })
```

Comment:
If we hide the element when you click delete, is it deleted only until we run the function again as it is still part of the array, can you think of how we might delete it by removing it from the array?